terraform/nifi.tf

```diff
@@ -0,0 +1,34 @@
+resource "aws_iam_user" "nifi" {
+  name = "bpm_nifi_email"
```

Comment:
do we want to add an environment to the name?

Reply:
Hmmm. We don't need to for AWS's sake, but it might help the TAs to tell them apart, okay, good idea.